src/Layers/xrRenderDX9/dx9HW.cpp

```diff
@@ -246,15 +246,27 @@ void CHW::Reset()
         DevPP.FullScreen_RefreshRateInHz = D3DPRESENT_RATE_DEFAULT;
     }
 
+    bool bNoManagedTex = !!RImplementation.o.managed_tex_disabled;
     while (true)
     {
-        const HRESULT result = HW.pDevice->Reset(&DevPP);
-        
+        HRESULT result;
+
+        if (bNoManagedTex)
+        {
+            GEnv.Render->ResourcesDeferredUnload();
```

Comment:
Bad place for this, should be done somewhere in `CRender` or `CResourceManager`.

Reply:
Will move to D3DXRenderBase::Reset()